### PR TITLE
fix: 🐛 `multiSig.getHistoricalProposal` errors

### PR DESCRIPTION
**Pull request by polymath-eric**
### Description

fixed GraphQL error when fetching historical proposals. Also add `stack` attribute to PolymeshError

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

File: src/api/entities/Account/MultiSig/index.ts [hunk 1]
```diff
@@ -157,26 +157,22 @@ export class MultiSig extends Account {
...
-    const { size, start } = opts;
+    const { context, address } = this;
+    const { size, start } = opts ?? {};
 
     const {
       data: {
         multiSigProposals: { nodes, totalCount },
       },
-    } = await queryMiddleware<Ensured<Query, 'multiSigProposals'>>(
+    } = await context.queryMiddleware<Ensured<Query, 'multiSigProposals'>>(
```
Comment (polymath-eric) [inline src/api/entities/Account/MultiSig/index.ts:175]: `this` was undefined in `queryMiddleware` when destructuring.

File: src/base/PolymeshError.ts [hunk 2]
```diff
@@ -24,14 +24,19 @@ export class PolymeshError extends Error {
     message,
     code,
     data,
+    stack,
   }: {
     message?: string;
     code: ErrorCode;
     data?: Record<string, unknown>;
+    stack?: Error['stack'];
```
Comment (polymath-eric) [inline src/base/PolymeshError.ts:32]: > Note: The stack property is de facto implemented by all major JavaScript engines, and [the JavaScript standards committee is looking to standardize it](https://github.com/tc39/proposal-error-stacks). You cannot rely on the precise content of the stack string due to implementation inconsistencies, but you can generally assume it exists and use it for debugging purposes.

~ [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/stack)

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**